fs/xfstests.py.data/README

```diff
@@ -0,0 +1,41 @@
+This is a simple wrapper for running xfstests inside avocado. The steps to get
+started are really simple:
+
+1) Edit the configuration variables on the yaml file as well environment variable
+1.1) The variables 'TEST_DEV' and 'TEST_DIR' are mandatory and should be set to
+     a block device path and mount point path, respectively, that will be used
+     *exclusively* for xfstests. It must have the filesystem of your choice
+     previously created.
+     ex : export TEST_DEV =/dev/vda1 , export TEST_DIR=/mnt
```

Comment:
@apahim i think it can be using export variable as of now user won't do mistake.In yaml case user may  use older entry  , i thinking when it pass through yaml want to have check on test case that device is valid or not so i kept this in TODO , what you think ?